### PR TITLE
SAK-45034 Webcomponents do not compile on windows

### PR DESCRIPTION
**Pull request by MRutea**
https://jira.sakaiproject.org/browse/SAK-45034

### CODE REVIEW

File: webcomponents/tool/src/main/frontend/.eslintrc.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ module.exports = {
   "rules": {
     "indent": ["warn", 2, { "SwitchCase": 1, "MemberExpression": "off", "ignoredNodes": ["TemplateLiteral > *"] }],
     "linebreak-style": ["error","unix"],
+    "linebreak-style": ["error","windows"],
```
Comment (adrianfish) [inline webcomponents/tool/src/main/frontend/.eslintrc.js:8]: That just breaks it on Linux and OSX and will break the CI build. Just change the unix one to "warn".

**Review by adrianfish — CHANGES_REQUESTED** [review]

File: webcomponents/tool/src/main/frontend/.eslintrc.js (hunk 1 above)
Comment (adrianfish) [reply webcomponents/tool/src/main/frontend/.eslintrc.js:8]: Also, read this: https://eslint.org/docs/rules/linebreak-style. Git is changing them to crlf on checkout, so the lint fails. But, "warn","unix" is fine, even if you don't update your git config.

Comment (MRutea) [reply webcomponents/tool/src/main/frontend/.eslintrc.js]: Hi, @adrianfish;
Thank you for the information.
I've changed "error" for "warn".

Greetings

**Review by adrianfish — APPROVED** [review]
GTG. Thanks.

**Pull request merged**